powerapps-docs/developer/model-driven-apps/troubleshoot-forms.md

```diff
@@ -389,4 +389,33 @@ The `XrmNavigation` operation in [Monitor](../../maker/model-driven-apps/monitor
 > ![XrmNavigation operation in Monitor](media/form-checker-navigation.png "XrmNavigation operation in [Monitor](../../maker/model-driven-apps/monitor-form-checker.md)")
 
 
+## Why is a different type of form opening instead of a Quick Create Form?
+
+When opening a QCF from a lookup or from a grid, a different type of form can open (Edit Form or Main Form Dialog). There are a few reasons for why this could happen.
```

Comment:
```suggestion
When opening a Quick Create Form (QCF) from a lookup or from a grid, a different type of form can open (Edit Form or Main Form Dialog). There are a few reasons for why this could happen.
```